molo/core/tests/test_views.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+import pytest
+
+
+@pytest.mark.django_db
+class TestPages(TestCase):
+    fixtures = ['molo/core/tests/fixtures/test.json']
```

Comment:
Does this continue to work when it's a package? Do we need to use pkg_resources perhaps? I've never actually thought of that before so I don't know.